Sources/docc/DocCDocumentation.docc/adding-tables-of-data.md

```diff
@@ -0,0 +1,160 @@
+# Adding Tables of Data
+
+Arrange information into rows and columns.
+
+## Overview
+
+To add a table to your documentation, start a new paragraph and add the following elements, each on its own line:
+
+* A table header, with the heading text of each column, separated by pipes (`|`)
+* A separator row, with at least 3 hyphens (`-`) for each column, separated by pipes
+* One or more rows of table cells, separating each cell of formatted content by a pipe
+ 
+```md
+Sloth speed  | Description                          
+------------ | ------------------------------------- 
+`slow`       | Moves slightly faster than a snail  
+`medium`     | Moves at an average speed           
+`fast`       | Moves faster than a hare            
+`supersonic` | Moves faster than the speed of sound
+```
+
+The example markup above defines the table that's shown below. Each column is automatically sized to fit its widest cell and the table is only as wide as the sum of its columns.
+
+Sloth speed  | Description                          
+------------ | ------------------------------------- 
+`slow`       | Moves slightly faster than a snail  
+`medium`     | Moves at an average speed           
+`fast`       | Moves faster than a hare            
+`supersonic` | Moves faster than the speed of sound
+
+It's not necessary to pad the cells to align the column separators (`|`). However, it may make your table _markup_ easier to read, especially for large or complex tables. 
+
+The same table could have been defined like below. All other examples will uses padded cells for readability.
+
+```md
+Sloth speed|Description
+---|---
+`slow`|Moves slightly faster than a snail
+`medium`|Moves at an average speed
+`fast`|Moves faster than a hare
+`supersonic`|Moves faster than the speed of sound
+```
+
+You can add leading and/or trailing pipes (`|`) if you find that table markup easier to read. This doesn't affect the rendered table on the page. The leading and trailing pipes _can_ be applied inconsistently for each row but doing so may make it harder to discern the structure of the table. 
+
+```md
+| Sloth speed  | Description                          |                         
+| ------------ | ------------------------------------ |
+| `slow`       | Moves slightly faster than a snail   | 
+| `medium`     | Moves at an average speed            |  
+| `fast`       | Moves faster than a hare             |
+| `supersonic` | Moves faster than the speed of sound |
+```
+
+The content of a table cell supports the same style attributes as other text and supports links to other content, including symbols. For more information about styling text, see [Format Text in Bold, Italics, and Code Voice](doc:formatting-your-documentation-content#Format-Text-in-Bold,-Italics,-and-Code-Voice).
+
+> Note: A table cell can only contain a single line of formatted content. Lists, asides, code blocks, headings, and directives are not supported inside a table cell. 
```

Comment:
Are there any inline directives that can appear inside a table cell? For example `@Image` ? My guess is that authors might want to display images inside of table cells often; whether this is supported or not we might want to mention that here.

Reply:
There are no inline directives at all. All directives are top-level "block" elements. This note already mentions that "directives are not supported inside a table cell"